manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -457,6 +457,8 @@ properties:
         keys:
           default:
             signingKey: (( grab secrets.uaa_jwt_signing_key ))
+          previous:
+            signingKey: (( grab secrets.uaa_jwt_signing_key_old || secrets.uaa_jwt_signing_key ))
```

Comment:
This line has a typo: it should be `secrets.uaa_jwt_signing_old_key`.

I only noticed this because both secrets were changed when rotating, which I wasn't expecting...

We should add a test for this.

Reply:
Well spotted! Feel free to push it back and I would fix this and write a test.